src/Network/Pusher/Base.hs

```diff
@@ -19,6 +20,23 @@ data ChannelInfo = ChannelInfo { occupied :: Bool
                                , subscriptionCount :: Maybe Int
                                } deriving Show
 
+
+data PartialChannel = PartialChannel { pUserCount :: Maybe Int } deriving Show
+
+instance FromJSON PartialChannel where
+  parseJSON (Object v) = PartialChannel <$> (v .:? "user_count")
+  parseJSON _ = mzero
+
+data Channel = Channel { name :: String, cUserCount :: Maybe Int } deriving Show
```

Comment:
My thinking was that the `Channel` data type is really what I want people interacting with, the others are just intermediaries. But you're right, `name` is too common, so I'll prefix.